icons/pixijs/pixijs-plain-wordmark.svg

```diff
@@ -0,0 +1 @@
5.004.121.008.192.16 1.762 1.645 3.113 3.426 3.191.078 0 .148.004.215.004zm42.496-.016c1.922-.144 3.441-1.71 3.457-3.624V57.69c-.008-1.89-1.559-3.433-3.465-3.449H85.148v34.215h9.891c.137 0 .281-.008.43-.02zm0 0"/></g><g clip-path="url(#b)" transform="matrix(1.00125 0 0 .99744 .008 .592)"><path fill="#e72264" d="M51.04 54.242h10.796c1.059 0 2.555.848 3.348 1.899l22.894 30.41c.79 1.05.574 1.902-.484 1.902H76.797c-1.059 0-2.559-.851-3.348-1.902L50.551 56.14c-.79-1.047-.57-1.899.484-1.899zm0 0"/></g><g clip-path="url(#c)" transform="matrix(1.00125 0 0 .99744 .008 .592)"><path fill="#e72264" d="M86.738 54.242H75.941c-1.058 0-2.554.848-3.347 1.899L49.699 86.55c-.793 1.05-.574 1.902.485 1.902H60.98c1.06 0 2.555-.851 3.348-1.902l22.895-30.41c.789-1.047.574-1.899-.485-1.899zm5.399-15.554c3.656 0 6.617 2.94 6.617 6.566 0 3.629-2.961 6.566-6.617 6.566-3.657 0-6.617-2.937-6.617-6.566 0-3.625 2.96-6.566 6.617-6.566zm-45.977 0c3.656 0 6.621 2.94 6.621 6.566 0 3.629-2.965 6.566-6.62 6.566-3.653 0-6.618-2.937-6.618-6.566 0-3.625 2.965-6.566 6.617-6.566zm74.887 0h-10.45c-3.75 0-6.792 3.019-6.792 6.742v10.367c0 3.719 3.043 6.738 6.793 6.738h10.449c3.75 0 6.793-3.02 6.793-6.738V45.43c0-3.723-3.043-6.742-6.793-6.742zm-6.902 6.8v6.996c0 .946-.192 1.762-.57 2.457a3.957 3.957 0 0 1-1.626 1.61c-.703.383-1.535.574-2.5.574-.816 0-1.523-.145-2.12-.438a4.26 4.26 0 0 1-1.509-1.226s-1.14-1.223-.097-2.086c1.066-.879 1.992.273 1.992.273.215.301.472.524.77.672a2.13 2.13 0 0 0 1 .227c.35 0 .66-.07.921-.215.266-.145.477-.36.633-.645.156-.285.234-.64.234-1.058v-7.14s-.027-1.442 1.415-1.442c1.44 0 1.457 1.441 1.457 1.441zm10.257 10.2c-.789.773-1.949 1.242-3.48 1.398-1.055.113-1.973.039-2.758-.215-.785-.254-1.516-.695-2.203-1.32 0 0-1.059-.906-.203-2.043.64-.856 1.844.035 1.844.035.457.422.949.73 1.472.937.524.204 1.114.27 1.774.2.586-.059 1.03-.22 1.34-.473.304-.254.437-.57.398-.953-.035-.332-.168-.594-.402-.785-.23-.192-.536-.344-.91-.461a13.051 13.051 0 0 0-1.215-.317c-.434-.093-.871-.21-1.317-.355a5.896 5.896 0 0 1-1.242-.574 3.074 3.074 0 0 1-.965-.965c-.262-.406-.426-.922-.492-1.55-.082-.786.035-1.477.352-2.075.32-.598.804-1.078 1.453-1.442.652-.363 1.41-.59 2.273-.68.899-.09 1.727-.023 2.492.212.762.23 1.41.586 1.946 1.054 0 0 .75.625.14 1.7-.472.832-1.797.332-1.797.332-.425-.352-.843-.602-1.25-.75-.41-.149-.851-.2-1.332-.149-.492.051-.87.184-1.14.395a.892.892 0 0 0-.344.84c.031.308.164.55.402.722.239.172.54.313.907.418a23.6 23.6 0 0 0 1.214.309c.438.097.883.215 1.325.355.445.14.859.34 1.234.602.379.262.703.601.973 1.015.265.418.437.958.507 1.622.125 1.203-.203 2.187-.996 2.96zM18.664 38.75C8.355 38.75 0 47.04 0 57.266c0 .199.004.398.012.597v19.61-.004 4.578c0 .035 0 .066.004.101v.082c.097 3.438 2.937 6.196 6.425 6.196 3.489 0 6.352-2.758 6.454-6.196v-6.566h5.332c.144.004.289.004.433.004 10.309 0 18.664-8.176 18.664-18.402 0-10.227-8.351-18.516-18.66-18.516zm0 24.133h-5.766V57.27c0-3.149 2.594-5.7 5.766-5.7s5.742 2.551 5.742 5.7c0 3.144-2.57 5.613-5.742 5.613zm0 0"/></g></svg>
```

Comment:
Since this icon is a plain version, it should only have a single `<path>` element. You can check out this guide for how to unify paths: https://github.com/devicons/devicon/issues/1306#issuecomment-1296014473

Make sure you optimize it using SVGOMG again after unifying the paths :)